test/subscription_manager.ts

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
There are no tests for, subscriptionExists, identityToKey, or genSubscriptionKey.